### PR TITLE
[CAM-9825] Change appliance name to 'Data Center Edition'

### PR DESCRIPTION
**Pull request by DiegoSanjuan**
in titles, flenames and references and add supported plarforms section to overview

### CODE REVIEW

**Comment by gsanchezu** [comment]
Hey Diego, please review the Travis build, which is failing: https://travis-ci.org/CenturyLinkCloud/PublicKB/builds/536888435?utm_source=github_status&utm_medium=notification

File: Cloud Application Manager/Datacenter Edition/camdce-initialsetup.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 {{{
 "title": "Setting Up Cloud Application Manager for Use",
```
Comment (gsanchezu) [inline Cloud Application Manager/Datacenter Edition/camdce-initialsetup.md]:
```suggestion
"title": "Setting Up Cloud Application Manager Data Center Edition for Use",
```

File: Cloud Application Manager/Datacenter Edition/camdce-migration.md [hunk 2]
```diff
@@ -1,12 +1,15 @@
...
 "title": "Deploy the New Appliance and Import the Data from the Prior Version",
-"date": "09-01-2016",
+"date": "05-24-2019",
 "author": "",
 "attachments": [],
 "contentIsHTML": false
 }}}
 
-### Deploy the New Appliance and Import the Data from the Prior Version
+### Deploy the New Datacenter Edition (appliance) and Import the Data from the Prior Version
```
Comment (gsanchezu) [inline Cloud Application Manager/Datacenter Edition/camdce-migration.md]: Please, add ToC, Overview, Audience and Prereqs as initial sections in all articles

File: Cloud Application Manager/Datacenter Edition/camdce-networking.md [hunk 3]
```diff
@@ -1,12 +1,12 @@
 {{{
 "title": "Configuring the Appliance Network Settings",
```
Comment (gsanchezu) [inline Cloud Application Manager/Datacenter Edition/camdce-networking.md]: Change this title as well

File: Cloud Application Manager/Datacenter Edition/camdce-overview.md [hunk 4]
```diff
@@ -1,29 +1,43 @@
...
 "contentIsHTML": false
 }}}
 
-### Cloud Application Manager Enterprise Appliance Overview
-Run Cloud Application Manager on-premise as a virtual appliance in your vCenter datacenter or OpenStack cloud. After you install and [set up Cloud Application Manager](appliance-initialsetup.md) for use, you’re ready to automate deployments to any cloud and platform.
+
+### Cloud Application Manager Datacenter Edition (appliance) Overview
+
+
+Run Cloud Application Manager on-premise as a virtual appliance in your vCenter datacenter or OpenStack cloud. After you install and [set up Cloud Application Manager](camdce-initialsetup.md) for use, you’re ready to automate deployments to any cloud and platform.
```
Comment (gsanchezu) [inline Cloud Application Manager/Datacenter Edition/camdce-overview.md]:
```suggestion
Run Cloud Application Manager on-premise as a virtual appliance in your vCenter datacenter or OpenStack cloud. After you install and [set up Cloud Application Manager Data Center Edition](camdce-initialsetup.md) for use, you’re ready to automate deployments to any cloud and platform.
```

File: Cloud Application Manager/Datacenter Edition/camdce-overview.md [hunk 5]
```diff
@@ -1,29 +1,43 @@
...
 **Try the Appliance for Free**
 The appliance is available as a free download. Experience all the same functionality as the paid Enterprise Edition, except for advanced features like managing your Cloud Application Manager organization, sharing and monitoring resources, and deploying to private clouds. To get these features, upgrade to the Enterprise Edition.
 **Download the Cloud Application Manager Virtual Appliance**
 Request a download link from [support](mailto:incident@CenturyLink.com).
 
+### Supported Platforms
+
+
+Appliance is supported when deployed on these platforms:
+* [Installing Appliance in vCenter](camdce-vsphere.md)
```
Comment (gsanchezu) [inline Cloud Application Manager/Datacenter Edition/camdce-overview.md]:
```suggestion
* [Installing Appliance in vCenter](camdce-vsphere.md)
```
```suggestion
* [Installing Cloud Application Manager Data Center Edition in vCenter](camdce-vsphere.md)
```
Please change al names below

File: Cloud Application Manager/Datacenter Edition/camdce-upgrading.md [hunk 6]
```diff
@@ -1,12 +1,12 @@
 {{{
 "title": "Upgrading the Appliance",
-"date": "09-01-2016",
+"date": "05-24-2019",
```
Comment (gsanchezu) [inline Cloud Application Manager/Datacenter Edition/camdce-upgrading.md]: Please, change title, author and keywords in all files

File: Cloud Application Manager/Datacenter Edition/camdce-with-ec2-instance-role.md [hunk 7]
```diff
@@ -104,7 +104,7 @@ In this example the policy is named *CAMApplianceAssumeRolePolicy*.
 
 ![AWS Console Role review](../../images/appliance/aws-assume-role-policy-role-review.png)
 
-### Attach the IAM Role to the Cloud Application Manager Data Center Edition EC2 Instance
+### Attach the IAM Role to the Cloud Application Manager Datacenter Edition EC2 Instance
```
Comment (gsanchezu) [inline Cloud Application Manager/Datacenter Edition/camdce-with-ec2-instance-role.md]: Please, revert this change and leave it as two words in all places

**Comment by gsanchezu** [comment]
Hey @jim-phillips, @JimGreene30, this PR is changing a whole section name, and several articles name, so please take that into account for redirects and so on. Thanks!

File: Cloud Application Manager/Datacenter Edition/camdce-migration.md [hunk 8]
```diff
@@ -1,12 +1,15 @@
...
+
+**Before starting upgrade process you should request an updated OVA to [contact support](mailto:incident@CenturyLink.com))**
```
[the remark this answers is not in the document]
Comment (DiegoSanjuan) [reply Cloud Application Manager/Datacenter Edition/camdce-migration.md]: done

File: Cloud Application Manager/Datacenter Edition/camdce-initialsetup.md [hunk 9]
```diff
@@ -1,13 +1,13 @@
...
-"date": "09-01-2016",
+"date": "05-24-2019",
 "author": "",
```
[the remark this answers is not in the document]
Comment (DiegoSanjuan) [reply Cloud Application Manager/Datacenter Edition/camdce-initialsetup.md]: done

File: Cloud Application Manager/Datacenter Edition/camdce-migration.md [hunk 10]
```diff
@@ -19,7 +22,7 @@ For the upgrade process to the new virtual appliance, please follow the steps be
    * When done, click **Save Settings**.
    **Note:** When switching the virtual disk, the appliance will reboot and become unavailable for a few minutes. This is expected as the existing data (appliance settings, logs, database and the most recent saved state) is being copied. **When the virtual appliance is back online, you can start using Cloud Application Manager**.
 
-3. Deploy the new appliance using this new [OVA](https://s3-us-west-1.amazonaws.com/elasticbox-appliance/2.0.124/ebx-appliance-2.0.124.ova) (without powering on).
+3. Deploy the new appliance using a new OVA (without powering on).
```
[the remark this answers is not in the document]
Comment (DiegoSanjuan) [reply Cloud Application Manager/Datacenter Edition/camdce-migration.md]: done

File: Cloud Application Manager/Datacenter Edition/camdce-networking.md [hunk 11]
```diff
@@ -1,12 +1,12 @@
...
-"date": "09-01-2016",
+"date": "05-24-2019",
```
[the remark this answers is not in the document]
Comment (DiegoSanjuan) [reply Cloud Application Manager/Datacenter Edition/camdce-networking.md]: done

File: Cloud Application Manager/Datacenter Edition/camdce-openstack.md [hunk 12]
```diff
@@ -1,12 +1,12 @@
 {{{
 "title": "Installing the Virtual Appliance in OpenStack",
-"date": "09-01-2016",
+"date": "05-24-2019",
```
[the remark this answers is not in the document]
Comment (DiegoSanjuan) [reply Cloud Application Manager/Datacenter Edition/camdce-openstack.md]: done

File: Cloud Application Manager/Datacenter Edition/camdce-overview.md (hunk 4 above)
Comment (DiegoSanjuan) [reply Cloud Application Manager/Datacenter Edition/camdce-overview.md]: done

File: Cloud Application Manager/Datacenter Edition/camdce-upgrading.md (hunk 6 above)
Comment (DiegoSanjuan) [reply Cloud Application Manager/Datacenter Edition/camdce-upgrading.md]: done

File: Cloud Application Manager/Datacenter Edition/camdce-overview.md (hunk 5 above)
Comment (DiegoSanjuan) [reply Cloud Application Manager/Datacenter Edition/camdce-overview.md]: done

File: Cloud Application Manager/Datacenter Edition/camdce-with-ec2-instance-role.md (hunk 7 above)
Comment (DiegoSanjuan) [reply Cloud Application Manager/Datacenter Edition/camdce-with-ec2-instance-role.md]: done

File: Cloud Application Manager/Administering Your Organization/admin-access.md [hunk 13]
```diff
@@ -45,7 +45,7 @@ All users with Cloud Application Manager organization administrator access.
 
 To revoke administrator privileges for a given user, just click the X button next to his or her name under **Administrators**.
 
-Administrators can manage several org-wide settings that include [user authentication](user-authentication.md), the [Cloud Application Manager appliance](../Appliance/appliance-overview.md), [provider access](provider-access.md), [admin boxes](../Automating Deployments/deploymentpolicy-box.md), [webhooks](webhooks.md), and [instance tags](resource-tags.md). To get an approach of the actions they can do, see [Overview](admin-overview.md).
+Administrators can manage several org-wide settings that include [user authentication](user-authentication.md), the [Cloud Application Manager Datacenter Edition](../Datacenter Edition/camdce-overview.md), [provider access](provider-access.md), [admin boxes](../Automating Deployments/deploymentpolicy-box.md), [webhooks](webhooks.md), and [instance tags](resource-tags.md). To get an approach of the actions they can do, see [Overview](admin-overview.md).
```
Comment (gsanchezu) [inline Cloud Application Manager/Administering Your Organization/admin-access.md]:
```suggestion
Administrators can manage several org-wide settings that include [user authentication](user-authentication.md), the [Cloud Application Manager Data Center Edition](../Data Center Edition/camdce-overview.md), [provider access](provider-access.md), [admin boxes](../Automating Deployments/deploymentpolicy-box.md), [webhooks](webhooks.md), and [instance tags](resource-tags.md). To get an approach of the actions they can do, see [Overview](admin-overview.md).
```

Comment (DiegoSanjuan) [reply Cloud Application Manager/Administering Your Organization/admin-access.md]: done

**Comment by gsanchezu** [comment]
Hey @jim-phillips, @JimGreene30, this PR is ready to be merged (it should be done before the push later today), but it is changing a whole section name, and several article names, so please take that into account for redirects and so on. Thanks!

**Comment by JimGreene30** [comment]
@jim-phillips, @gsanchezu -- I'll get the redirects setup. Thanks!
